### ISSUE TITLE
Should bobbin gdb behave differently with j-link?

### ISSUE THREAD

**Issue by kjetilkjeka** [bobbin-rs/bobbin-cli#3]
When calling ```bobbin gdb``` with j-link you must first start the GDB server and set port manually. Wouldn't it be more user-friendly if JLinkGDBServer was started in a child process? This way one could have the port specified in the config, and make sure the server started on the same port as gdb eventually would connect to.

This would perhaps also remove the need for the ```bobbin jlink``` command, making the interface more uniformly over all debug hardware.

**Comment by jcsoo** [comment]
Yes, that was the original intended behavior, but I ran into some problems where Control-C signals going to GDB were also being sent to the JLinkGDBServer / OpenOCD process, killing them. Doing this properly requires setting up a separate process group for the GDB server when spawning the child. I will be taking a look into this.
